example/newrepo/main.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2018 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// The newrepo command utilizes go-github as a cli tool for
+// creating new repositories.
+
```

Comment:
Please remove this blank line so that Godoc can show the above comment as the package's main documentation.